GliaWidgetsTests/Sources/EngagementLauncher/EngagementLauncherTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+@testable import GliaWidgets
+
+final class EngagementLauncherTests: XCTestCase {
+
+    func test_startChat() throws {
+        var chatEngagement: EngagementKind?
+        var engagementLauncher = EngagementLauncher { engagementKind, _ in
```

Comment:
Why this is instantiated on its own? Does integrator instantiate `EngagementLauncher` like that? As I understand there is dedicated method on the SDK to get `EngagementLauncher` instance, so it seems logically to use it. This will make sure that property `Glia.engagement` has correct value, making the test much deeper. Does it make sense? Also the name of the test can provide more information on what is tested.

Reply:
Good catch. So this tests remains, because EngagementLauncher here as a unit. I added tests for get/start engagement launcher in scope of Glia.

https://github.com/salemove/ios-sdk-widgets/pull/1055/commits/b9452a76d2e2c1b46170b675b1431e9750bcc66d